src/SFA.DAS.ApprenticeCommitments.Jobs/ApprenticeshipUpdated.cs

```diff
@@ -4,8 +4,8 @@ namespace SFA.DAS.ApprenticeCommitments.Jobs
 {
     public class ApprenticeshipUpdated
     {
-        public long ApprenticeshipId { get; set; }
-        public DateTime ApprovedOn { get; set; }
-        public string Email { get; set; }
+        public long? ContinuationOfCommitmentsApprenticeshipId { get; set; }
```

Comment:
Seeing as we're changing the name anyway, would it be possible to modify it a touch further to make it easier to understand?  "Continuation of apprenticeship ID" suggests we're extending the ID, not the apprenticeship 🤷 

Maybe `ContinuedCommitmentsApprenticeshipId`, or anything else that makes grammatical sense.

Reply:
I don't mind want we call it, but in Approvals it's called ContinuationOfId and the name here mirrors that (in a  manner). However if you have a suggestion for a more appropriate name. I'll update it in each of the layers